app/src/main/java/es/wolfi/app/passman/CredentialEdit.java

```diff
@@ -187,8 +205,22 @@ public void onSuccess(int statusCode, cz.msebera.android.httpclient.Header[] hea
                                 && fileObject.has("size") && fileObject.has("created") && fileObject.has("mimetype")) {
                             fileObject.put("filename", fileName);
                             File file = new File(fileObject);
-                            fed.addFile(file);
-                            fed.notifyDataSetChanged();
+
+                            if (requestCode == 2) {
+                                fed.addFile(file);
+                                fed.notifyDataSetChanged();
+                            }
+                            if (requestCode == 3) {
+                                JSONObject customFieldJSONObject = new JSONObject();
```

Comment:
I believe this kind of data handling would be better placed on the [api library](https://github.com/nextcloud/passman-android/tree/master/app/src/main/java/es/wolfi/passman/API)

Reply:
fixed in 5b250f8317c18764586a3d11ea130c0c944160c7